### DISCUSSION TITLE
Go to definition opens search for definition.

### DISCUSSION THREAD

**Discussion by Ssionn** [zed-industries/zed#13775] [Feature Requests]
I have this issue where my editor is opening up search for definition when I want it to directly go there, anyway to turn this off?

https://github.com/zed-industries/zed/assets/90753599/54116520-b19c-476d-ad7b-5973d32f3179

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
